HISTORY.rst

```diff
@@ -3,6 +3,17 @@
 Release History
 ===============
 
+0.11.2
```

Comment:
The `0.11.1` release hasn't been published yet, but due to the scope of changes we will aim for `0.12.0` as the next IoT extension release.

In this PR, it would be nice to combine these `0.11.2` notes with the `0.11.1` notes below, and change the version label to `0.12.0`. But we can take care of that later/not strictly required in this PR.

Also we'll work to improve the versioning process in future iterations.

Reply:
Fixed.